### ISSUE TITLE
Calling x509_hasClosePrimes without second argument?

### ISSUE THREAD

**Issue by aarongable** [crtsh/certwatch_db#82]
I'm not very familiar with postgres' function programming, so this may not be a bug, but it seemed suspect enough that I wanted to report it just in case.

The new `x509_hasClosePrimes` function is called in the exact same way as the existing `x509_hasROCAFingerprint` function: with the single certificate (or rather, a pointer to it) as its only argument:
https://github.com/crtsh/certwatch_db/blob/1c29538b838ea06d83271b0e3ef863c7d012b4b4/fnc/web_apis.fnc#L1718-L1719

However, the function itself accesses a second argument, the number of iterations of Fermat's factorization that it should perform:
https://github.com/crtsh/libx509pq/blob/b91e554a9a9415a6b60157656f3eff8e2e1328af/libx509pq.c#L3482

It seems likely that this second argument access is getting a default value of some sort (I'd guess 0, except that the check [appears to be running at least once](https://crt.sh/?id=5048682432)) or is reading uninitialized memory? If it's working as intended, I'd love to know where that second argument is coming from!

**Comment by robstradling** [comment]
@aarongable Thanks for raising this, Aaron.  When I created the function prototype on the primary crt.sh DB I specified a default value of 100 for the second parameter.  This default value was committed to our internal version control repository, but somehow it didn't make it to https://github.com/crtsh/libx509pq.

Fixed by https://github.com/crtsh/libx509pq/commit/e43fcab8dea3c6e238a72f6cbbf1541c2bb26503
